docs/en/docs-nav.json

```diff
@@ -112,6 +112,23 @@
             {
               "text": "10: Book to Author Relation",
               "path": "tutorials/book-store/part-10.md"
+            },
```

Comment:
Do not put this under the bookstore tutorial. Instead, add jus before the community articles (as the last tutorial)

Reply:
here: 
![image](https://github.com/user-attachments/assets/b2f82e0a-5ee0-43ac-902e-09dcf5526e17)